packages/sdk/src/sdk/services/Encryption/EmailEncryptionService.ts

```diff
@@ -219,4 +242,146 @@ export class EmailEncryptionService extends BaseAPI {
     const json = await response.json()
     return base64.decode(json.data)
   }
+
+  /**
+   * Process and encrypt emails in batches for backfill
+   * @param adminUserId The admin user ID to use as grantor
+   * @param purchases The purchase records to process
+   * @returns Array of encrypted data records
+   */
+  async processAndEncryptEmails(
```

Comment:
this is temporary, will be pulled out after backfill is complete

Reply:
does this method get called anywhere?

Reply:
just on my local client to generate the CSV, just tossing here so it doesn't get lost, will remove once the complete migration is finished

Reply:
could leave as a gh comment so you don't have to revert later but sounds good